components/service/fxa/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
+   - License, v. 2.0. If a copy of the MPL was not distributed with this
+   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
+<manifest package="org.mozilla.fxa"
```

Comment:
Following the pattern of our other libraries this should be: `mozilla.components.service.fxa` (Telemetry is a special case currently as it existed before)